.gitignore

```diff
@@ -10,7 +10,6 @@ dist/
 *.log
 
 #cypress
-reports/
```

Comment:
Removed this, as it was not showing the changes that are happening in the reports folder in git diff.